video/out/gpu/video.c

```diff
@@ -384,7 +384,13 @@ const struct m_sub_options gl_video_conf = {
             {"hable",    TONE_MAPPING_HABLE},
             {"gamma",    TONE_MAPPING_GAMMA},
             {"linear",   TONE_MAPPING_LINEAR},
-            {"bt.2390",  TONE_MAPPING_BT_2390})},
+            {"bt.2390",  TONE_MAPPING_BT_2390},
+            // Experimental #HDR on #macOS        
+            // Added two tone mapping modes
+            // HDR Passthrough - render HDR content based on peak level
+            {"hdrpass",  TONE_MAPPING_HDR_PASSTHROUGH},
+            // HDR Scale - render HDR content and ignore peak level, this way any content can become exended
+            {"hdrscale", TONE_MAPPING_HDR_SCALE})},
```

Comment:
These should not be tone mapping curves. Rather, tone mapping should be disabled for true HDR output. (Unless the source peak exceeds the HDR output's target peak, in which case BT.2390 should be used)

Reply:
Do you think a new switch should be added then?